src/oc_bifrost/apps/bifrost/priv/pgsql_statements.config

```diff
@@ -53,6 +53,24 @@
       WHERE authz_id=$1
     ) AS exists">>}.
 
+{get_actor_id,
+<<"SELECT id FROM auth_actor WHERE authz_id=$1">>}.
```

Comment:
These don't feel right.  id is a database internal identifier.  It's a bigserial, not a char(32).  It isn't supposed to be exposed outside the db if my intuition about how these schemas are designed is right.